libs/utils/env.py

```diff
@@ -600,6 +601,14 @@ def _init_target_gem5(self):
             system['platform']['description']))[0]
         self.conf['board'] = self.conf['board'].lower() + suffix
 
+        energy_file = self.conf['board'] + '_energy';
+        try:
+            self.nrg_model = globals()[energy_file]
```

Comment:
Without digging any deeper I don't see a clean solution for that, however I have a comment: maybe we could change the way platform files are loaded, so that gem5 can use the same files as the "real" platforms ? I understand that in some cases you may still want a gem5-specific file, so maybe we could prioritize them (gem5-specific file first, otherwise real platform file).